### PR TITLE
add diagrams to how kv works

### PR DESCRIPTION
**Pull request by koeninger**

### CODE REVIEW

File: content/workers/learning/how-kv-works.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ weight: 7
 
 Workers KV is a global, low-latency, key-value data store. It stores data in a small number of centralized data centers, then caches that data in Cloudflare's data centers after access. KV supports exceptionally high read volumes with low latency, making it possible to build highly dynamic APIs and websites that respond as quickly as a cached static file would. While reads are periodically revalidated in the background, requests which are not in cache and need to hit the centralized back end can see high latencies.
 
+When you write to KV, your data is written to central data stores. It is not sent automatically to every location’s cache.
+
+![Writing to KV](/workers/static/kv-write.svg)
+
+Initial reads from a location do not have a cached value. The data must be read from the nearest central data store, resulting in a slower response.
+
+![Cache misses](/workers/static/kv-slow-read.svg)
```
Comment (deadlypants1973) [inline content/workers/learning/how-kv-works.md]:
```suggestion
![Initial reads will miss the cache and go to the nearest central data store first.](/workers/static/kv-slow-read.svg)
```

File: content/workers/learning/how-kv-works.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ weight: 7
...
+
+Frequent reads from the same location return the cached value without reading from a central data store, resulting in faster response times.
+
+![Cache hits](/workers/static/kv-fast-read.svg)
```
Comment (deadlypants1973) [inline content/workers/learning/how-kv-works.md]:
```suggestion
![As mentioned above, frequent reads will return a cached value.](/workers/static/kv-fast-read.svg)
```

File: content/workers/learning/how-kv-works.md [hunk 3]
```diff
@@ -8,6 +8,18 @@ weight: 7
 
 Workers KV is a global, low-latency, key-value data store. It stores data in a small number of centralized data centers, then caches that data in Cloudflare's data centers after access. KV supports exceptionally high read volumes with low latency, making it possible to build highly dynamic APIs and websites that respond as quickly as a cached static file would. While reads are periodically revalidated in the background, requests which are not in cache and need to hit the centralized back end can see high latencies.
 
+When you write to KV, your data is written to central data stores. It is not sent automatically to every location’s cache.
+
+![Writing to KV](/workers/static/kv-write.svg)
```
Comment (deadlypants1973) [inline content/workers/learning/how-kv-works.md]:
```suggestion
![Your data is written to central data stores when you write to KV.](/workers/static/kv-write.svg)
```

**Review by deadlypants1973 — APPROVED** [review]
Approved with small suggestions for image alt text

File: content/workers/learning/how-kv-works.md (hunk 2 above)
Comment (koeninger) [reply content/workers/learning/how-kv-works.md]: Aren't these just going to result in someone with a screen reader hearing essentially the same sentence twice (paragraph and then alt text)?

Comment (deadlypants1973) [reply content/workers/learning/how-kv-works.md]: You are correct. I tried to rewrite it in a way that encompasses what happens in the image but couldn't figure out a sentence with the words "cache hit" in my brain. The principle I was taught was that images need to have a description comprised of a full sentence that flows with the rest of the article, almost as if an image isn't there, which is why I wanted to change it to a full sentence. https://usability.yale.edu/web-accessibility/articles/images#image-guidelines

**Review by rts-rob — APPROVED** [review]
LGTM

**Pull request merged**